support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -224,16 +224,39 @@ function getAmountsTestParticipations(
 		return urlTest;
 	}
 
-	const { test } = settings.amounts?.[countryGroupId] ?? {};
+	if (!settings.amounts) {
+		return null;
+	}
+
+	const AMOUNTS = settings.amounts;
+
+	// Two-tier amounts - check for country test, then region test
+	let testArray = AMOUNTS.filter((t) => t.target === country);
```

Comment:
I think `find` is the way to go here, but if you want to stick with `filter` you could always use array destructuring to get the first matched test, to avoid having to index into and array later, e.g.

```js
const [matchedTest] = AMOUNTS.filter((t) => t.target === country);
```

Reply:
Will go with find, as in the other PRs

Reply:
Code updated